migration/migrating-4_2-4/migrating-application-workloads-4_2-to-4.adoc

```diff
@@ -1,12 +1,12 @@
 [id="migrating-application-workloads-openshift-4_2-to-4"]
-= Migrating application workloads between {product-title} 4.2 clusters
+= Migrating application workloads from {product-title} 4.2 to {product-version}
 include::modules/common-attributes.adoc[]
 :context: migrating-4_2-4_x
 :migrating-4_2-4_x:
 
 toc::[]
 
-You can migrate application workloads between {product-title} 4.2 clusters with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+You can migrate application workloads between {product-title} 4.2 to {product-version} with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
```

Comment:
Should "between" be "from" now here?

Reply:
Yes! Good catch. Thought I'd fixed that. Will do